README.md

```diff
@@ -12,27 +14,13 @@ and performance.
 Thanks to [Browserstack](https://www.browserstack.com/) for providing access to
 their great cross-browser testing tools.
 
-## Table of contents
-
-* [Introduction](#introduction)
-* [Features](#features)
-* [Inline example](#inline-example)
-* [Examples](#examples)
-* [Core documentation](#core-documentation)
-* [Modules documentation](#modules-documentation)
-* [Helpers](#helpers)
-* [Virtual Node documentation](#virtual-node)
-* [Structuring applications](#structuring-applications)
```

Comment:
Moved the TOC to after the links to more examples. This way there's some introductory content before a long table of contents.